README.md

```diff
@@ -0,0 +1,45 @@
+# process-compose-flake
+Based on [process-compose](https://github.com/F1bonacc1/process-compose)
+
+## Usage
+Let's say you want to have a devShell which makes a command `watch-server` available, that you can use to spin up your projects `backend-server`, `frontend-server` and `proxy-server`.
+
+To achieve this using `process-compose-flake` you can simply add the following code to the `perSystem` function in your `flake-parts` flake.
+```
+process-compose.configs = {
+  watch-server.processes = {
+    backend-server.command = "${self'.apps.backend-server.program} --port 9000";
+    frontend-server.command = "${self'.apps.frontend-server.program} --port 9001";
+    proxy-server.command =
```

Comment:
@roberth This is a flake-parts module to mimic the process-compose part of https://devenv.sh/processes/

Do you have any thoughts on the API convention here? In particular the "configs" naming.